packages/blade/src/components/Icons/UpiAutopayFilledIcon/UpiAutopayFilledIcon.tsx

```diff
@@ -0,0 +1,49 @@
+import { Svg, Path } from '../_Svg';
```

Comment:
UPI or Upi

Reply:
we already have UpiAutoPayIcon, can't change this